content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,113 @@
+
+This identifies this Location as a "Virtual" one, used to create a hierarchical structure for
+a Warehouse, aggregating its "Child" Locations.
+
+.. note::
+   Products can *not* be stored in this :guilabel:`Location Type`.
+
+.. image:: stock_warehouses/stock-warehouses-location-types.png
    :align: center
+   :alt: Warehouse location types in location creation screen.
+
+Configure physical warehouse locations
+--------------------------------------
 
-Sell a product from the virtual warehouse
-=========================================
+Navigate back to the :guilabel:`Locations` overview (via the breadcrumbs), then click into the
+first physical :guilabel:`Warehouse` stock :guilabel:`Location`.
 
-Let’s say you have two products, one stored in warehouse A and one stored in warehouse B. Now, you
-can create a new quotation for one of each product. Go to other information and choose Warehouse A+B
-in the shipping information.
+Under :guilabel:`Parent Location`, select the virtual :guilabel:`Warehouse` from the drop-down
+menu. Then, navigate back to the :guilabel:`Locations` overview, and repeat this step for the
+second physical :guilabel:`Warehouse` stock :guilabel:`Location`.
 
-.. image:: stock_warehouses/ware3.png
+Both Locations are now "Child" Locations of the virtual warehouse "Parent" location.
+
+.. tip::
+   To use a virtual "Parent" Location as the default Warehouse for sales orders, each salesperson
+   can have the Warehouse assigned to them from the drop-down menu next to
+   :guilabel:`Default Warehouse` on their employee form.
+
+.. image:: stock_warehouses/stock-warehouses-employee-form.png
    :align: center
+   :alt: Default warehouse location on employee form.
+
+Example flow: Sell a product from a virtual warehouse
+=====================================================
+
+To sell products from multiple Warehouses using a virtual "Parent" Location, there must be at least
+*two* products and at least *two* Warehouses configured - with at least *one* product in each
+Warehouse.
+
+To create a new :abbr:`RFQ (Request for Quotation)`, navigate to the :guilabel:`Sales` app, and
+click :guilabel:`Create`. Fill out the information on the new quotation by adding a
+:guilabel:`Customer`, and click :guilabel:`Add a product` to add the two products stored in the two
+Warehouses.
 
-Once you have done it, you can convert it to a sales order. Then, a delivery order will be
-automatically generated, with a product reserved in warehouse A and one in warehouse B.
+Then, click the :guilabel:`Other Info` tab, and change the :guilabel:`Warehouse` to the virtual
+Warehouse that was previously created. Once the quotation has been filled out, :guilabel:`Confirm`.
 
-.. image:: stock_warehouses/ware4.png
+Now that the quotation has been changed to a sales order, click the :guilabel:`Delivery` smart
+button. From here, change the :guilabel:`Source Location` to the virtual :guilabel:`Warehouse` that
```

Comment:
If you create the order with a user that has the virtual warehouse assigned do we still need to make all these changes on the delivery order?

This seems like a lot of extra steps (almost a workaround and not a feature to reduce time)

Reply:
I showed this "long way 'round" method on purpose, to show that it can be done even if somebody is just doing it for the first time. I **did** write a `.. tip::`, however, saying that you can shorten this process by doing exactly what you suggested. I just moved that to the bottom of the page where it fits better!